server/api/controllers/userController.js

```diff
@@ -6,22 +6,28 @@ import User from '../models/user';
 import createToken from '../helpers/authToken';
 
 class Users{
-	static signup(req, res) {
+	static async signup(req, res) {
 		// eslint-disable-next-line camelcase
 		const {
-			first_name, last_name, email, password,
+			email, first_name, last_name, password,
 		} = req.body;
 		const newUser = User.createNewUser({
-			first_name, last_name, email, password,
+			email, first_name, last_name, password,
```

Comment:
Are you hashing the password?

Reply:
No, my passwords are not hashed

Reply:
You have to because your database can be leaked and the hacker you shouldn't be able to know what was the password. Sometimes even your know employees can manually access the database to get user's passwords if they are not hashed (encrypted).

Reply:
You can use [bcrypt](https://www.npmjs.com/package/bcrypt) to do the hashing